{
  try {
    set.verify();
  }
 catch (  final AssertionFailedError e) {
    return;
  }
  fail();
}
